os_net_config/cli.py

```diff
@@ -276,20 +280,51 @@ def main(argv=sys.argv, main_logger=None):
         main_logger.debug("Interface report requested, exiting after report.")
         print(json.dumps(reported_nics))
         return retval
-    try:
-        iface_array = get_iface_config(
-            "network_config",
-            opts.config_file,
-            iface_mapping,
-            persist_mapping,
-            strict_validate=opts.exit_on_validation_errors,
-        )
-    except objects.InvalidConfigException as e:
-        main_logger.error("Schema validation failed for network_config\n%s", e)
-        return ExitCode.ERROR
 
-    if not iface_array:
-        return ExitCode.ERROR
+    config_dict = {
```

Comment:
section = {"remove_config": [],
                  "network_config": [],
...
}
should do I guess.